internal/cursorediterator/blocks_test.go

```diff
@@ -994,7 +995,7 @@ func TestCursoredProducerMapperIterator(t *testing.T) {
 					return func(yield func(ItemAndCursor[int], error) bool) {
 						for i, item := range chunk {
 							mu.Lock()
-							processingOrder = append(processingOrder, fmt.Sprintf("mapper: processing %s", item))
+							processingOrder = append(processingOrder, "mapper: processing "+item)
```

Comment:
This is a preference, but I don't like string concatenation when format strings would work in a given context. How big is the performance difference? Can we disable this particular rule?